javascript/javascript2/week3/functionVariable.js

```diff
@@ -0,0 +1,54 @@
+/*
+Function as a variable
+
+Create funtions that are used in these different ways:
+*/
+
+//**************************************************************************************
+// 1- Create an array with 3 items. All items should be functions. Iterate through the array and call all the functions.
+//**************************************************************************************
+const funcArray = [funct1(), funct2(), funct3()];
+
+function funct1() {
+  console.log(`This is function 1`);
+}
+
+function funct2() {
+  console.log(`This is function 2`);
+}
+
+function funct3() {
+  console.log(`This is function 3`);
+}
+
+funcArray.forEach((element) => {
+  return element;
+});
```

Comment:
In const funcArray = [funct1(), funct2(), funct3()]; you are executing functions,
so 
funcArray.forEach((element) => {
  return element;
}); is not doing anything that is shown in the console.

The better way of doing this task is to have an array that contains only function's names (without executing), and in forEach loop you can call them;